### PR TITLE
Fix/UI style

### PR DESCRIPTION
**Pull request by bobetbat**
Search page responsiveness 
https://simard.atlassian.net/jira/software/projects/WIN/boards/3?selectedIssue=WIN-518

### CODE REVIEW

File: src/components/MapBox.tsx [hunk 1]
```diff
@@ -286,7 +286,7 @@ export const MapBox: React.FC = () => {
                     <Popup>
                       <SearchCard
                         key={f.id}
-                        sm={true}
+                        mapCard={true}
```
Comment (silksil) [inline src/components/MapBox.tsx:289]: if i read `mapCard` i don't know what it happening in the searchCard. Is it showing a mapCard, or?

Can you make the naming more descriptive?

File: src/components/SearchCard.tsx [hunk 2]
```diff
@@ -75,23 +75,28 @@ export const SearchCard = forwardRef<HTMLDivElement, SearchCardProps>(
...
               media={facility.media}
             />
           </Stack>
           <Stack
             onClick={() => navigate(`/facility/${facility.id}`)}
             justifyContent="space-between"
-            width={sm ? '252px' : '100%'}
-            spacing={1}
-            sx={{ py: 2, px: 1.5, mt: 0, cursor: 'pointer' }}
+            width={small ? theme.spacing(32) : '100%'}
```
Comment (silksil) [inline src/components/SearchCard.tsx:89]: in the whole app, we handle responsive different then here, which makes it harder to read. Also we are creating exepcetions for naming, for example you use `small` but throughout the app we use the breakpoints `xs` or `sm`. 

You can either do this to make it responsive:

```const StackStyled = styled(Stack)(({ theme }) => ({
  width: theme.spacing(32),

  [theme.breakpoints.up('md')]: {
   width: "100%",
  }
}));```

Or this:

```
<Stack
  sx={{ width: { xs: theme.spacing(32), md: "100%"}}}

File: src/components/SearchCard.tsx [hunk 3]
```diff
@@ -125,15 +130,15 @@ export const SearchCard = forwardRef<HTMLDivElement, SearchCardProps>(
...
-              direction={winWidth < 900 || sm ? 'row-reverse' : 'row'}
+              direction={medium || mapCard ? 'row-reverse' : 'row'}
               alignItems="end"
-              justifyContent={winWidth < 900 || sm ? 'start' : 'space-between'}
+              justifyContent={'space-between'}
             >
               <Stack
-                direction={winWidth < 900 || sm ? 'column' : 'row'}
-                alignItems={winWidth < 900 || sm ? 'end' : 'center'}
+                direction={medium || mapCard ? 'column' : 'row'}
```
Comment (silksil) [inline src/components/SearchCard.tsx:138]: same as the other comment about responsiveness

File: src/components/MapBox.tsx (hunk 1 above)
Comment (bobetbat) [reply src/components/MapBox.tsx]: card on the map

File: src/components/SearchCard.tsx (hunk 2 above)
Comment (bobetbat) [reply src/components/SearchCard.tsx]: outdated

File: src/components/Results.tsx [hunk 4]
```diff
@@ -107,56 +97,65 @@ export const Results: React.FC = () => {
...
-          className="noScrollBar"
-        >
+      <StyledContainer className="noScrollBar">
+        {showResultsNumber && (
+          <Stack direction="column" alignItems="center" justifyContent="center">
+            <Box
+              sx={{
+                justifySelf: 'center',
+                alignSelf: 'center',
+                background: 'black',
```
Comment (silksil) [inline src/components/Results.tsx:134]: black is hardcoded, is this correct?

File: src/hooks/useWindowScrollPositions.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react';
+
+export const useWindowScrollPositions = () => {
```
Comment (silksil) [inline src/hooks/useWindowScrollPositions.ts:3]: sweet!

File: src/components/MapBox.tsx [hunk 6]
```diff
@@ -273,26 +313,31 @@ export const MapBox: React.FC = () => {
...
-                      <SearchCard
-                        key={f.id}
-                        sm={true}
-                        facility={f}
-                        isSelected={f.id === selectedFacilityId}
-                        numberOfDays={numberOfDays}
-                        focusedEvent={f.eventInfo}
-                      />
-                    </Popup>
+                    {isMobileView ? null : (
```
Comment (silksil) [inline src/components/MapBox.tsx:328]: in the `SearchForm` it is called `isMobile` and here it is called `isMobileView` 

to keep it consistent, i suggest to pick one name.

File: src/components/MapBox.tsx (hunk 1 above)
Comment (silksil) [reply src/components/MapBox.tsx]: so this shows it? and if it is `fals`e it doesn't show the map on the card?

If so, can you make it descriptive in the naming that is shown or not shown, e.g. `showMapCard` or `isMapCardShown` ?

File: src/components/MapBox.tsx [hunk 7]
```diff
@@ -127,6 +150,9 @@ export const MapBox: React.FC = () => {
   const { selectedFacilityId } = useAppState();
   const dispatch = useAppDispatch();
 
+  const theme = useTheme();
+  const isMobileView = useMediaQuery(theme.breakpoints.down('md'));
```
Comment (silksil) [inline src/components/MapBox.tsx:154]: in the `SearchForm` it is refered to as `isMobile` and here it is called `isMobileView`. In `SearchCard` it is called `medium`. They all use the same breakpoints. 

Let's pick one name to keep it consistent.

File: src/components/MapBox.tsx (hunk 6 above)
Comment (silksil) [inline src/components/MapBox.tsx:328]: ```isMobileView &&  ( ...``` should also be possible

File: src/components/SearchCard.tsx (hunk 3 above)
Comment (silksil) [reply src/components/SearchCard.tsx:130]: I would personallly stick as much as possible to the way MUI recommends us to use responsiveness in styling properties. This  prevents naming inconsistencies (e.g. you call it `medium` here but `isMobile` on a different page.

``` direction={{ xs: 'column', md: mapCard ? 'column' : 'row}}```

ofc you sometime have to use the `useMediaQuery` hook like you are doing here, but I would not do that for styling . Only to include/exclude a component on certain screen sizes.

File: src/components/MapBox.tsx (hunk 1 above)
Comment (silksil) [reply src/components/MapBox.tsx]: `MapCard` and `Results` are still in the components folder, but should be moved to the container folder.

File: src/components/MapBox.tsx (hunk 6 above)
Comment (bobetbat) [reply src/components/MapBox.tsx:328]: arrived from main

File: src/components/MapBox.tsx (hunk 7 above)
Comment (bobetbat) [reply src/components/MapBox.tsx:154]: arrived from main

File: src/components/Results.tsx (hunk 4 above)
Comment (bobetbat) [reply src/components/Results.tsx:134]: arrived from main

File: src/components/MapBox.tsx (hunk 6 above)
Comment (bobetbat) [reply src/components/MapBox.tsx:328]: isMobileView is incorrect its tablet 900 is tablet or medium

**Review by silksil — APPROVED** [review]

**Pull request merged**